gsa/src/web/pages/performance/performancepage.js

```diff
@@ -393,10 +386,10 @@ const mapStateToProps = (rootState, {gmp}) => {
 
   if (gmp.settings.enableGreenboneSensor) {
     SLAVE_SCANNER_FILTER = Filter.fromString(
-      'type=' + GMP_SCANNER_TYPE + ' type=' + GREENBONE_SENSOR_SCANNER_TYPE,
+      'type=' + GREENBONE_SENSOR_SCANNER_TYPE,
     );
   } else {
-    SLAVE_SCANNER_FILTER = Filter.fromString('type=' + GMP_SCANNER_TYPE);
+    SLAVE_SCANNER_FILTER = Filter.fromString('');
```

Comment:
I would expect this filter to return all scanners and I think we want to return none in this case, right? I might be wrong, though.

Reply:
The whole else clause should be removed.